README.md

```diff
@@ -1,7 +1,7 @@
 
 # imperialb.in
 
-> A hastebin alternative but with user experience in mind with much more features!
+## A hastebin alternative but with user experience in mind and much more features!
```

Comment:
actually can you change this to IMPERIAL is a code/text storing site where you can share, edit, or encrypt documents with integrations such as Discord and GitHub Gists.

Reply:
yes I can

Reply:
fixed